web/src/test/java/com/epam/rft/atsy/cucumber/login/LoginStepDefs.java

```diff
@@ -40,9 +40,9 @@ public void bejelentkezesButtonClicked() {
     driverProvider.getDriver().findElement(By.id("loginButton")).click();
   }
 
-  @Then("the Candidates page appears")
+  @Then("the Candidates page appears and user logs out")
```

Comment:
Adding the log out action to this can be dangerous, I don't recommend to include this solution into the master branch.